hazelcast/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -114,8 +114,8 @@
 
 @SuppressWarnings({"unused", "MismatchedQueryAndUpdateOfCollection"})
 @Command(
-        name = "jet",
-        description = "Utility to perform operations on a Hazelcast Jet cluster.%n" +
+        name = "platform",
+        description = "Utility to perform operations on a Hazelcast Platform cluster.%n" +
```

Comment:
I've heard that the new merged thing will be called just "Hazelcast". Is the `name` used anywhere on command line? Can't we remove it completely?
```suggestion
        name = "hazelcast",
        description = "Utility to perform operations on a Hazelcast cluster.%n" +
```

Reply:
It is fixed after merging master into this branch.